files/es/mdn/writing_guidelines/page_structures/macros/commonly_used_macros/index.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Macros usadas comunmente
+slug: MDN/Writing_guidelines/Page_structures/Macros/Commonly_used_macros
+tags:
+  - CSS
+  - Estructuras
+  - Macros
+  - Referencia
+translation_of: MDN/Structures/Macros/Commonly-used_macros
+original_slug: MDN/Structures/Macros/Commonly-used_macros
+---
+{{MDNSidebar}}
+
+Esta página enumera muchas de las macros de propósito general creadas para usarlas en MDN. Para obtener información sobre cómo usar estas macros, consulta {{web.link("/es/docs/MDN/Contribute/Structures/Macros", "Uso de macros")}} y {{web.link("/es/docs/MDN/Contribute/Editor/Links#Usar_macros_de_enlaces", "Usar macros de enlaces")}}. Consulta {{web.link("/es/docs/MDN/Contribute/Structures/Macros/Other", "Otras macros")}} para obtener información sobre las macros que se utilizan con poca frecuencia, que se utilizan solo en contextos especiales o, están en desuso. También hay una {{web.link("/es/dashboards/macros", "lista completa de todas las macros en MDN")}}.
+
+Consulta también la {{web.link("/es/docs/MDN/Contribute/Guidelines/CSS_style_guide", "guía de estilo CSS")}} para conocer los estilos disponibles para usarlos.
+
+## Enlaces
+
+### Creando un solo hipervínculo
+
+En general, no es necesario utilizar macros para crear enlaces arbitrarios. Utiliza el botón **Enlace** en la interfaz del editor para crear enlaces.
+
+- La macro {{TemplateLink("Glossary")}} crea un vínculo a la entrada de un término específico en el {{web.link("/es/docs/Glossary", "glosario")}} de MDN. Esta macro acepta un parámetro obligatorio y dos opcionales:
+
+  Ejemplos:
+
+  1.  El nombre del término (tal como "HTML").
+  2.  El texto que se mostrará en el artículo en lugar del nombre del término (esto se debe usar con poca frecuencia).{{Optional_Inline}}
+  3.  Si se especifica este parámetro y no es cero, no se aplica el estilo personalizado que normalmente se aplica a los enlaces del glosario.{{Optional_Inline}}
+
+  - `\{{Glossary("HTML")}}` produce {{Glossary("HTML")}}
+  - `\{{Glossary("CSS", "Hojas de estilo en cascada")}}` produce {{Glossary("CSS", "Hojas de estilo en cascada")}}
+  - `\{{Glossary("HTML", "", 1)}}` produce {{Glossary("HTML", "", 1)}}
+
+- {{TemplateLink("anch")}} — crea un enlace a una sección en la página:
+
+  - `\Enlace a páginas en referencias`;
```

Comment:
Missing link

```suggestion
  - `\[Enlace a páginas en referencias](/es/docs/MDN/Writing_guidelines/Page_structures/Macros/Commonly_used_macros#enlace_a_p%C3%A1ginas_en_referencias)`;
```

Reply:
Ack, this is an example of the `anch` macro, which has been removed -- let's just remove this part entirely.